resource-aggregate/events/resourceStateSnapshotTaken.go

```diff
@@ -381,6 +411,46 @@ func (e *ResourceStateSnapshotTaken) HandleCommand(ctx context.Context, cmd aggr
 			return nil, err
 		}
 		return []eventstore.Event{&rc}, nil
+	case *commands.CreateResourceRequest:
+		if req.GetCommandMetadata() == nil {
+			return nil, status.Errorf(codes.InvalidArgument, errInvalidCommandMetadata)
+		}
+
+		em := MakeEventMeta(req.GetCommandMetadata().GetConnectionId(), req.GetCommandMetadata().GetSequence(), newVersion)
```

Comment:
I would propose to validate if this resource is of type `oic.wk.collection`

Reply:
At the resource with don't know resource type because this values are stored at /plgd/res resource.